src/redux/actions/orgUnitDistReport.js

```diff
@@ -0,0 +1,71 @@
+import i18n from '../../utils/i18n/locales'
+import { getOrgUnitDistReport } from '../../utils/api'
+import { showSuccessSnackBar, showErrorSnackBar } from './feedback'
+import {
+    loadingReportDataStart,
+    loadingReportDataSuccessWithFeedback,
+    loadingReportDataErrorWithFeedback,
+} from './reportData'
+
+export const actionTypes = {
+    CHART_IMAGE_URL_LOADING_START: 'LOADING_CHART_IMAGE_URL_START',
+    CHART_IMAGE_URL_LOADING_SUCCESS: 'LOADING_CHART_IMAGE_URL_SUCCESS',
+    CHART_IMAGE_URL_LOADING_ERROR: 'LOADING_CHART_IMAGE_URL_ERROR',
+}
+
+/**
+ * @returns {Object}
+ */
+export const loadingChartImageUrlStart = () => ({
+    type: actionTypes.CHART_IMAGE_URL_LOADING_START,
+})
+
+/**
+ * @param {string} chartImageUrl
+ * @returns {Object}
+ */
+export const loadingChartImageUrlSuccess = () => ({
```

Comment:
This part will have to be discussed with Lars. I'm not sure if getting an image is still the way to go now. Good chance we just need to transform the JSON data and feed it to a chart using Chart.js. There is no mention of getting an image in the [webapi docs](https://docs.dhis2.org/master/en/developer/html/dhis2_developer_manual_full.html#webapi_org_unit_distribution)